planet/clients/orders.py

```diff
@@ -422,6 +422,13 @@ async def list_orders(self, state: str = None, limit: int = None):
         """
         url = self._orders_url()
 
+        # Set default limit to 100
+        if limit is None:
+            limit = 100
+        # Set no limit
+        elif limit == 0:
+            limit = None
```

Comment:
If we change the function definition to `list_orders(self, state: str = None, limit: int = 100)` then we can delete lines 425-427 and avoid the limit/None switcheroo.

Reply:
Jeez ya, that's way better 🥇